contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -1,28 +1,18 @@
-doc: | 
+category: application
+doc: |
   This is the most general application definition for multidimensional
   photoelectron spectroscopy.
-category: application
-uri: http://purl.org/pan-science/PaNET/PaNET01269
-NXmpes:
+type: group
+NXmpes(NXobject):
   (NXentry):
     title:
     start_time(NX_DATE_TIME):
       doc: |
         Datetime of the start of the measurement.
-    end_time(NX_DATE_TIME):
```

Comment:
As you can see here the changes are also reflected in the `yaml` file. So somehow an old version slipped into this branch.

Reply:
Fixed, now using the correct yaml version in mpes-refactor.